### PR TITLE
docs(readme): Update README for Add SSO

### PR DESCRIPTION
**Pull request by MuyangAmigo**
To align with content in Wiki

### CODE REVIEW

**Review by KennethBWSong — APPROVED** [review]

File: packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md [hunk 1]
```diff
@@ -1,17 +1,85 @@
-## How to enable SSO in TeamsFx Bot projects
+# Enable single sign-on for bot applications
```
Comment (KennethBWSong) [inline packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md:1]: Shall we mention that this doc is for latest version?

Comment (MuyangAmigo) [reply packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md:1]: You mean this doc is for the latest toolkit version?

Comment (KennethBWSong) [reply packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md:1]: Yes, we may mention this doc is for extension >= xxx

Comment (MuyangAmigo) [reply packages/fx-core/templates/plugins/resource/aad/auth/bot/README.md:1]: This readme is scaffolded by us, probably not necessary to me.

**Review by KennethBWSong — APPROVED** [review]

**Pull request merged**